docs/api/conventions.md

```diff
@@ -81,6 +81,30 @@ The path to the server build, relative to remix.config.js. Defaults to "build".
 
 The port number to use for the dev server. Defaults to 8002.
 
+### importMapFile
+
+A path to an [import map](https://github.com/WICG/import-maps), relative to remix.config.js. This is used at build time to resolve imports. If [bundleImportMap](#bundleimportmap) is "true", the import paths must be HTTPS URL's.
```

Comment:
Can we be more specific here about when this is actually used? Something like:

A path to an import map (relative to remix.config.js) that is used at build time to resolve imports in the browser build. Also used by the Deno server runtime to resolve imports.

Is that accurate? Too soon to talk about Deno support?

Reply:
This is nothing to do with deno. The server build doesn't include this. Deno will use an import map and re-map bare module imports that way totally out of the scope of remix.